docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. The jdbc table is table in jdbc complaint database. To provide parallelism during read each instance of Apex input operator also known as operator partition reads one portion of jdbc table. Based on requested operator instances number of non-polling static partitions are created to scan table portitions. The non-polling partition will scan the table portion only once. Additional single operator partition is created to scan newly added data incrementally. 
+
+#### Assumption
+Assumption is that there is an ordered column using which range queries can be formed. That means database has a column which has unique constraint as well as the newly inserted record should have column value more than max value in that column.
+
+## Use cases
+1. Scan huge database tables to either copy to other database or process it using **Apache Apex**. The example application to copy database contents to HDFS is available in [examples repository](https://github.com/DataTorrent/examples/tree/master/tutorials/jdbcIngest)
+
```

Comment:
We also have a variety of operators named "AbstractJdbc*" under lib/db/jdbc/. How should the reader go about
deciding which operator to use ?

"The example application" => "An example application using this operator"
"available in" => "available in the"